package.json

```diff
@@ -27,7 +27,8 @@
     "basic-logger": "^0.4.4",
     "dive": "^0.5.0",
     "dotenv": "^16.0.3",
-    "mongodb": "^5.1.0"
+    "mongodb": "^5.1.0",
+    "typescript": "^4.2.3"
```

Comment:
I think `typescript` is generally advocated to be a dev dependency. I don't think it matters too much in the grand scheme of things, but I'm wondering if it would be good practice to instead update the Dockerfile to have a separate build stage where we can install dev dependencies and build.

If this seems like a lot of work and is better done another time, this is fine with me

Reply:
good call @rayangler ! see latest commit

Reply:
Oh, sorry, I was thinking about [Docker's stages](https://docs.docker.com/build/building/multi-stage/#use-multi-stage-builds)! Similar to how the [Autobuilder's](https://github.com/mongodb/docs-worker-pool/blob/master/Dockerfile) has a building stage. I have an in-progress smaller scale [example](https://github.com/10gen/snooty-data-api/blob/b11aa2f6e9d69c8001ca3b47bb27e537ab08ffc1/Dockerfile) for the Snooty Data API.

Reply:
TIL docker 😎